### PR TITLE
Added section on INSPIRE

### PR DESCRIPTION
**Pull request by alexanderkotsev**
New section on INSPIRE reusing content from the previously existing (OSGeoLive 10.5) section on standards.

### CODE REVIEW

**Comment by cvvergara** [comment]
@alexanderkotsev 
We think that this document is going to be better placed on the overviews on a new section we will create.
Its a bit long process to do that, and
A) I can guide you through the process (look for me on osgeolive irc)
B) I do the process myself

You decide.

**Comment by alexanderkotsev** [comment]
Thanks @cvvergara . I would like to give it a try with Option A.

File: doc/CMakeLists.txt [hunk 1]
```diff
@@ -5,6 +5,7 @@
 #---------------------
 set(OSGeoLiveDoc_FILES
     #"disclaimer.rst"
+    "standards.rst"
```
Comment (cvvergara) [inline doc/CMakeLists.txt]: Remove this line

File: doc/_themes/overview/page.html [hunk 2]
```diff
@@ -25,6 +25,7 @@
             <ul id="top-nav">
               <li><a href="../index.html">Home</a></li>
               <li><a href="../overview/overview.html">Contents</a></li>
+              <li><a href="../standards.html">Standards</a></li>
```
Comment (cvvergara) [inline doc/_themes/overview/page.html]: Remove this line

File: doc/_themes/overview/page.html [hunk 3]
```diff
@@ -43,6 +44,7 @@
             <ul id="top-nav">
               <li><a href="./index.html">Home</a></li>
               <li><a href="./overview/overview.html">Contents</a></li>
+              <li><a href="./standards.html">Standards</a></li>
```
Comment (cvvergara) [inline doc/_themes/overview/page.html]: Remove this line

File: doc/index.rst [hunk 4]
```diff
@@ -10,6 +10,7 @@ Welcome to |osgeolive-version|
 
      overview/toc
      quickstart/toc
+     standards
```
Comment (cvvergara) [inline doc/index.rst]: Remove this line

File: doc/standards.rst [hunk 5]
```diff
@@ -0,0 +1,50 @@
+Geospatial standards
```
Comment (cvvergara) [inline doc/standards.rst]: move this file
`git mv doc/standards.rst doc/overview/inspire_overview.rst`

**Comment by cvvergara** [comment]
Closing in favor of #411

**Pull request closed without merging**